lib/cn/csched_sp3.c

```diff
@@ -1524,127 +1525,38 @@ sp3_job_serialize(struct sts_job *const job, void *const arg)
     tm = (jclock_ns - w->cw_t0_enqueue) / NSEC_PER_SEC;
     snprintf(tmbuf, sizeof(tmbuf), "%lu:%02lu", (tm / 60) % 60, tm % 60);
 
-    if (ev(!cJSON_AddNumberToObject(node, "cnid", w->cw_tree->cnid))) {
+    bad = !cJSON_AddNumberToObject(node, "cnid", w->cw_tree->cnid);
+    bad = !cJSON_AddNumberToObject(node, "node", w->cw_node->tn_nodeid);
```

Comment:
If an earlier cJSON call fails and any of the subsequent ones succeed doesn't this overwrite bad?

Reply:
Heh!  He meant to type |=

Reply:
yep...